
uint8 STOPPED=0
uint8 RUNNING=1

int32 state